src/tools/sse-tools.ts

```
@@ -0,0 +1,125 @@
+ * Handles configuration of the SSE transport
+ */
+export async function configureSSE(args: SSEConfigArgs): Promise<ServerResult> {
+  try {
+    const { action, port, path } = args;
+    const config = await configManager.getConfig();
+    let result = "";
+
+    switch (action) {
+      case "status":
+        // Return current SSE configuration status
+        const enabled = config.sseEnabled === true;
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)